nncf/tensorflow/layers/wrapper.py

```diff
@@ -202,6 +203,15 @@ def _apply_ops(self, training):
             self.set_layer_weight(weight_attr, layer_weight)
 
     def registry_weight_operation(self, weights_attr: str, op: NNCFOperation):
+        # For BatchNormalization layers:
+        # If layer.scale=False then gamma parameter is fused - no need to apply the mask to it
+        if isinstance(self.layer, tf.keras.layers.BatchNormalization) \
```

Comment:
You should use layer metatypes or add SyncBatchNormalization to comparison because there are actually two layers with this behavior.

Reply:
Thanks for this comment. I will use layer metatypes.